param/optical_flow_params_sim.yaml

```diff
@@ -0,0 +1,54 @@
+# Settings for the optical flow estimator
+optical_flow_estimator:
+    # fov of the camera
+    fov: 1.00
+
+    # Minimum altitude to run the detector
+    min_estimation_altitude: 0.15
+
+    # Window size for optical flow
+    win_size: 20
+
+    # Levels of the pyramid to run the optical flow at
+    max_level: 3
+
+    # Number of iterations to run the optical flow
+    iters: 20
+
+    # Number of points to track
+    points: 300
+
+    # Quality level for the corner detector
+    quality_level: 0.01
+
+    # Min distance between points for the corner detector
+    min_dist: 15
+
+    # Image scale factor
+    scale_factor: 0.3
+
+    # Debug settings for the optical flow estimator
+    # Publish image with the velocity vectors drawn on it
+    debug_vectors_image: true
+
+    # Publish image with average velocity vector drawn on it
+    debug_average_vector_image: true
+
+    # Print out times when things are received/processed
+    debug_times: false
+
+    # Min variance to send with velocity measurements
+    variance: 0.1
+
+    # What to scale the variance on according to the rotation rate
+    variance_scale: 1.0
+
+    # Cutoff area where velocity vectors aren't included in the average
```

Comment:
What does this mean?

Reply:
Percentages of the image to ignore, see `findAverageVector` in `OpticalFlowEstimator.hpp`